hnn_core/params.py

```diff
@@ -177,15 +177,15 @@ def _extract_drive_specs_from_hnn_params(params, cellname_list):
             if cname_ampa in par:
                 ampa_weight = par[cname_ampa][0]
                 ampa_delay = par[cname_ampa][1]
-                if ampa_weight > 0.:
+                if ampa_weight >= 0.:
```

Comment:
Since most param files contain zero-weighted drive connection parameters, this allows these param files to still be instantiated in hnn-core without throwing an error in drives.py L33-35.

Reply:
Could you update `whats_new.rst` so we know what changed from a user perspective?